main.nf

```diff
@@ -473,7 +474,8 @@ process sortExtractMapped{
   set val(name), file('*_mapped.fq.gz') into reads_mapped
 
   script:
-  def collate_fast = params.samtools_collate_fast ? "-f -r 100000" : ""
+  def rim = params.reads_in_memory ? params.reads_in_memory : "10000"
```

Comment:
Since `reads_in_memory` defaults to `10000` as set in the nextflow.config, this should always evaluate to true, no? So I think you can omit this and in the line below directly access `params.reads_in_memory` instead

Reply:
Same for all the other processes

Reply:
Also the `default` went down by a factor of 10 compared to before, was that intentional?

Reply:
Ah yes, you‘re right. I‘ll change it in the nextflow.config then. 
Setting it to 10000 was intentional